p2p/simulations/network_test.go

```diff
@@ -158,3 +378,79 @@ func triggerChecks(ctx context.Context, ids []enode.ID, trigger chan enode.ID, i
 		}
 	}
 }
+
+// \todo: refactor to implement shapshots
+// and connect configuration methods once these are moved from
+// swarm/network/simulations/connect.go
+func BenchmarkMinimalService(b *testing.B) {
+	b.Run("ring/32", benchmarkMinimalServiceTmp)
+}
+
+func benchmarkMinimalServiceTmp(b *testing.B) {
+
+	// stop timer to discard setup time pollution
+	b.StopTimer()
+	args := strings.Split(b.Name(), "/")
+	nodeCount, err := strconv.ParseInt(args[2], 10, 16)
+	if err != nil {
+		b.Fatal(err)
+	}
+
+	for i := 0; i < b.N; i++ {
```

Comment:
Do we need `b.StopTimer()` as well as the first line in the for loop for iterations `i > 0`?

Reply:
To rephrase, it would be good not to measure the first part in the for loop body (until the line 432) in interactions `i > 0`. Currently, it is excluded only in the first iteration.

Reply:
Right you are. Well spotted!